docs/development.md

```diff
@@ -42,6 +49,16 @@ Use Ctrl-C to stop
 
 Default username and password for admin are: `admin@example.com` and `test123`.
 
+#### App with storefront generated by the gem
+
+To create a sandboxed Solidus application where the storefront is generated with
+this gem, run `GENERATE_FRONTEND=1 bin/sandbox`. The path for the sandbox app
+is `./sandbox-generated`. Unlike the previous `./sandbox` app, you'll need to
+`cd` to the `./sandbox-generated` directory in order to run the application.
```

Comment:
Do you think we could use an env var to dispatch to `sandbox-generated` when given?

Reply:
@waiting-for-dev You mean, something like `FORWARD=sandbox-generated bin/rails`? I think we can update `bin/rails` to support that:

```rb
#!/usr/bin/env ruby

if %w[g generate].include? ARGV.first
  exec "#{__dir__}/rails-engine", *ARGV
elsif ['sandbox', 'sandbox-generated'].include?(ENV['FORWARD'])
  exec "#{__dir__}/rails-#{ENV['FORWARD']}", *ARGV
elsif ENV['FORWARD'].present?
  raise "Unknown value for FORWARD environment variable '#{ENV['FORWARD']}'"
else
  exec "#{__dir__}/rails-sandbox", *ARGV
end
```

What do you think?

Reply:
Yeah, I mean doing something like that. However, we could lower the cognitive burden if we use the same environment variable to generate the sandbox app and forward it to it through `bin/rails`. What do you think if we use `GENERATED` for both `GENERATE_FRONTEND` and `FORWARD`? I really don't matter about the name that much as long as it's the same for both scenarios.

Reply:
@waiting-for-dev Please see https://github.com/nebulab/solidus_starter_frontend/pull/171#discussion_r685716524.